ETC.tsv

```diff
@@ -7062,10 +7062,10 @@ ETC_20150317_007450	Suspicious spell device
 ETC_20150317_007451	Activate spell device
 ETC_20150317_007453	Hire Mercenary!
 ETC_20150317_007454	Protect big merchant Gilliam
-ETC_20150317_007455	There is a news that a big merchant Gilliam who did not sell items to any person has been placed at a crossroad of new city.
+ETC_20150317_007455	There are news that Gilliam, a big merchant who did not sell items to anyone, has been placed at a crossroad of a new city.
```

Comment:
Change There are to There is. Change has been placed at a crossroad to "has been placed at the crossroad of a new city."